### ISSUE TITLE
Stashes based on branches don't show up if the branches got deleted

### ISSUE THREAD

**Issue by cateyes99** [mhutchie/vscode-git-graph#266]
**Describe the Bug**
Stashes based on branches don't show up in **Git Graph**, if those branches got deleted.

**Steps to Reproduce**
Steps to reproduce the behaviour:
1. Create a new branch, make a few commits into this branch (in the example it's called `12204_remove-hardcoded-annualPayment-logic`).
2. Make some more changes, and stash them on the new branch.
3. Check **Git Graph**, you can see the new stash: `stash@{0}` of `b90183c3d4424ccb35af413b1fd8f9d44b4588f2`.
4. Delete this new branch.
5. Check **Git Graph** again, you should be able to see the new stash: `stash@{0}`. But actually now you don't see `stash@{0}` now **_:(_**

**Expected Behaviour**
At the Reproduce Step 5, `stash@{0}` should still display in **Git Graph**.

**Environment**
 - Git Graph Extension Version: _1.21.0_
 - Visual Studio Code Version: _1.41.1_
 - Operating System: _Windows 10 Professional_

**Screenshots**
>λ git log --graph --date-order --oneline                                                           
```
* commit a786b53beac1d6fdceca80a88f6bba7f01d00758 (HEAD -> master, origin/master, origin/HEAD)
| Author: John Doe
| Date:   Sun Feb 23 20:05:16 2020 +0000
|
|     Merged PR 12980: Remove the hardcoded annual payment logic
|
|     Remove the hardcoded annual payment logic
|
|     Workitem #16957 #16958
|
|     Related work items: #16957, #16958
|
| *   commit b90183c3d4424ccb35af413b1fd8f9d44b4588f2 (refs/stash)
| |\  Merge: 5bcae686 6f2d0911
| | | Author: John Doe
| | | Date:   Sat Feb 22 21:15:38 2020 +1300
| | |
| | |     On 12204_remove-hardcoded-annualPayment-logic: Commented out brand claims check
| | |
| | * commit 6f2d0911a1f959470d79de83bf79b43395cce749
| |/  Author: John Doe
| |   Date:   Sat Feb 22 21:15:38 2020 +1300
| |
| |       index on 12204_remove-hardcoded-annualPayment-logic: 5bcae686 Fix the unit tests
| |
| * commit 5bcae686e61e429b9c67fa02abb04c41fe7a1552
| | Author: John Doe
| | Date:   Mon Feb 10 11:26:33 2020 +1300
| |
| |     Fix the unit tests
| |
| |     Workitem #16957
| |
| * commit 3d1c5faa5a9b7d86acc01d338350b14955664369
|/  Author: John Doe
|   Date:   Mon Feb 10 10:03:02 2020 +1300
|
|       Remove the hardcoded annual payment logic
|
|       Workitem #16957
|
* commit 6e0d6793e4a142ad4de1055c978f4955c6a40adf
| Author: John Doe
| Date:   Fri Feb 21 02:02:35 2020 +0000
|
|     Merged PR 13150: add logging
|
|     Updated PoliciesService.cs
|
|     Related work items: #16417
|
* commit 4c706f4cf48891a31364ee759b509948091d5611
| Author: John Doe
| Date:   Thu Feb 20 20:41:07 2020 +0000
|
|     Merged PR 13130: new mapping for janet and rich
|
* commit 3ba53521cdbb6bdb9cc589a8ca6b6f0416db592a
| Author: John Doe
```
There is another `stash@{1}` which was stashed based on the **master** branch quite a while ago. As the master branch is always there, so this `stash@{1}` still displays itself in **Git Graph** as shown below.
![image](https://user-images.githubusercontent.com/19678455/75307060-a476ed80-58af-11ea-808f-3e03bf0ba854.png)

**Comment by mhutchie** [comment]
Thanks for spotting this, I’ll have a fix for this available in the next few days.

**Comment by mhutchie** [comment]
This will be available in v1.22.0.

If you'd like to use it before the next release, you can download [v1.22.0-beta.4](https://github.com/mhutchie/vscode-git-graph/releases/tag/v1.22.0-beta.4), and install it following the instructions provided [here](https://github.com/mhutchie/vscode-git-graph/wiki/Downloading-&-Installing-Releases#manual-installation).

**Comment by cateyes99** [comment]
Thanks, it's good now 👍
